packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
+export default {
+  inheritAttrs: false,
+};
+</script>
+<script lang="ts" setup>
+import { computed, toRefs, type PropType, reactive } from 'vue';
+import {
+  SfScrollableDirection,
+  SfScrollableButtonsPlacement,
+  useScrollable,
+  SfIconChevronLeft,
+  SfIconChevronRight,
+  SfButton,
+  type SfScrollableOnDragChangeData,
+  type SfScrollableOnScrollData,
+  type SfScrollableOnPrevData,
+  type SfScrollableOnNextData,
+  ClassProp,
+} from '@storefront-ui/vue';
+
+const props = defineProps({
+  tag: {
+    type: String,
+    default: 'div',
+  },
+  direction: {
+    type: String as PropType<`${SfScrollableDirection}`>,
+    default: SfScrollableDirection.horizontal,
+  },
+  buttonsPlacement: {
+    type: String as PropType<`${SfScrollableButtonsPlacement}`>,
+    default: SfScrollableButtonsPlacement.block,
+  },
+  wrapperClass: ClassProp,
+  activeIndex: {
+    type: Number,
+    default: undefined,
+  },
+  reduceMotion: {
+    type: Boolean,
+    default: undefined,
+  },
+  drag: {
+    type: Boolean,
+    default: undefined,
+  },
+  previousDisabled: {
+    type: Boolean,
+    default: undefined,
+  },
+  nextDisabled: {
+    type: Boolean,
+    default: undefined,
+  },
+});
+const emit = defineEmits<{
+  (e: 'onDragChange', data: SfScrollableOnDragChangeData): void;
+  (e: 'onScroll', data: SfScrollableOnScrollData): void;
+  (e: 'onPrev', data: SfScrollableOnPrevData): void;
+  (e: 'onNext', data: SfScrollableOnNextData): void;
+}>();
+const { direction, activeIndex, reduceMotion, drag } = toRefs(props);
+
+const { getContainerRef, state, getNextButtonProps, getPrevButtonProps } = useScrollable(
+  computed(() => ({
+    ...reactive({
+      direction,
+      activeIndex,
+      reduceMotion,
+      drag,
+    }),
+    onDragChange: (data) => emit('onDragChange', data),
+    onScroll: (data) => emit('onScroll', data),
+    onPrev: (data) => emit('onPrev', data),
+    onNext: (data) => emit('onNext', data),
+  })),
+);
+
+const changeDisabledClass = (isDisabled: boolean) =>
+  isDisabled ? '!ring-gray-200 !text-disabled-500' : '!ring-neutral-500 !text-neutral-500';
+const isHorizontal = computed(() => props.direction === SfScrollableDirection.horizontal);
+</script>
+
+<template>
+  <div :class="['items-center', 'relative', isHorizontal ? 'flex' : 'flex-col h-full inline-flex', wrapperClass]">
+    <div v-if="$slots.previouButton" v-bind="getPrevButtonProps"><slot name="previouButton" /></div>
```

Comment:
typo, should be `prevousButton`, here and below 
```suggestion
    <div v-if="$slots.previousButton" v-bind="getPrevButtonProps"><slot name="previousButton" /></div>
```